### PR TITLE
Resolving an issue in php run images by allowing casing in php file extension

### PR DESCRIPTION
**Pull request by arroyc**
this pr is to solve the following problem that customer reported

> All of our PHP images are no longer able to handle the “PHP” extension if it’s not all in lower case (i.e. index.php).  If the name is in any other format such as camel or uppercase (i.e. index.pHp, index.PHP, index.pHP), we’re exposing the whole file

### CODE REVIEW

File: tests/Oryx.RuntimeImage.Tests/Oryx.RuntimeImage.Tests.csproj [hunk 1]
```diff
@@ -24,6 +24,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\..\src\BuildScriptGenerator\BuildScriptGenerator.csproj" />
+    <ProjectReference Include="..\Oryx.BuildImage.Tests\Oryx.BuildImage.Tests.csproj" />
```
Comment (kichalla) [inline tests/Oryx.RuntimeImage.Tests/Oryx.RuntimeImage.Tests.csproj]: Why is this being referenced? Kind of weird to reference BuildImage tests project in runtimeimages test

File: tests/Oryx.RuntimeImage.Tests/Php/PhpImageTest.cs [hunk 2]
```diff
@@ -65,6 +85,77 @@ public void GraphicsExtension_Gd_IsInstalled(string imageTag)
             Assert.True((bool)((JValue)gdInfo.GetValue("PNG Support")).Value);
         }
 
+        [Theory]
+        [InlineData("7.3")]
+        [InlineData("7.2")]
+        [InlineData("7.0")]
+        [InlineData("5.6")]
+        public async Task Check_If_Apache_IsConfigured_For_PHP(string imageTag)
```
Comment (kichalla) [inline tests/Oryx.RuntimeImage.Tests/Php/PhpImageTest.cs]: Rename the test to the pivot being tested i.e the casing of the file extension

File: tests/Oryx.RuntimeImage.Tests/Oryx.RuntimeImage.Tests.csproj (hunk 1 above)
Comment (arroyc) [reply tests/Oryx.RuntimeImage.Tests/Oryx.RuntimeImage.Tests.csproj]: it's not needed, I will clean it .. thanks

File: tests/Oryx.RuntimeImage.Tests/Php/PhpImageTest.cs (hunk 2 above)
Comment (arroyc) [reply tests/Oryx.RuntimeImage.Tests/Php/PhpImageTest.cs]: sure

File: tests/Oryx.RuntimeImage.Tests/Php/PhpImageTest.cs [hunk 3]
```diff
@@ -65,6 +85,77 @@ public void GraphicsExtension_Gd_IsInstalled(string imageTag)
             Assert.True((bool)((JValue)gdInfo.GetValue("PNG Support")).Value);
         }
 
+        [Theory]
+        [InlineData("7.3")]
+        [InlineData("7.2")]
+        [InlineData("7.0")]
+        [InlineData("5.6")]
+        public async Task Check_If_Apache_Allows_Casing_In_PHP_File_Extension(string imageTag)
```
Comment (kichalla) [inline tests/Oryx.RuntimeImage.Tests/Php/PhpImageTest.cs]: ApacheServer_DoesNotServePhpFilesAsRawText_ForDifferentCasingOfFileExtension or something? (pivot here is to make sure Apache does not serve as raw text). Nitty thing as good test names can make code readable. You need not handle this comment in this PR.

**Pull request merged**